### ISSUE TITLE
Add documentation

### ISSUE THREAD

**Issue by cmfcmf** [lmm-git/Beam#17]
I'd like to try this module, but there isn't any documentation!

**Comment by lmm-git** [comment]
We are working on the module and then on the documentation. I will inform you when we start to write the documentation and when you can test the module.

**Comment by cmfcmf** [comment]
Did you start writing documentation yet? :book: :books:

**Comment by lmm-git** [comment]
Yes and we implemented the plugin-system more (this needs a lot further documentation). Now I am only looking for the best desktop environment (currently we use Unity and thats not vary fast and flexible...)
